amm/repl/src/main/scala/ammonite/main/Defaults.scala

```diff
@@ -25,32 +25,34 @@ object Defaults{
   // Need to import stuff from ammonite.ops manually, rather than from the
   // ammonite.ops.Extensions bundle, because otherwise they result in ambiguous
   // imports if someone else imports maunally
-  val predefString = s"""
-    |import ammonite.ops.{
-    |  PipeableImplicit,
-    |  FilterMapExtImplicit,
-    |  FilterMapArraysImplicit,
-    |  FilterMapIteratorsImplicit,
-    |  FilterMapGeneratorsImplicit,
-    |  SeqFactoryFunc,
-    |  RegexContextMaker,
-    |  Callable1Implicit
-    |}
-    |import ammonite.runtime.tools._
-    |import ammonite.repl.tools._
-    |import ammonite.interp.api.IvyConstructor.{ArtifactIdExt, GroupIdExt}
-    |import ammonite.interp.api.InterpBridge.value.exit
-    |""".stripMargin
+  val predefImports = Imports(
+    ImportData("ammonite.interp.api.InterpBridge.value.exit"),
+    ImportData("ammonite.interp.api.IvyConstructor.{ArtifactIdExt, GroupIdExt}", importType = ImportData.Type),
+    ImportData("""ammonite.ops.{
+      PipeableImplicit,
+      FilterMapExtImplicit,
+      FilterMapArraysImplicit,
+      FilterMapIteratorsImplicit,
+      FilterMapGeneratorsImplicit,
+      SeqFactoryFunc,
+      RegexContextMaker,
+      Callable1Implicit
```

Comment:
IMO We should probably kill most of these as well. They were a cute experiment, but I've been moving away from the DSL-y operator-y style of ammonite.ops for most of my code, and this is a natural time to drop these imports. Anyone who wants them can get them by `import ammonite.ops._` anyway, which is no great hardship

Reply:
I'm all for that, let's try!